dashboard/test/controllers/api/v1/sections_controller_test.rb

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+require 'test_helper'
+
+class Api::V1::SectionsControllerTest < ActionController::TestCase
+  self.use_transactional_test_case = true
```

Comment:
Yes, it's required for `setup_all`, which cuts down significantly on test time over `setup` with this many creates.